runtime/src/currency_id_convert.rs

```diff
@@ -1,64 +1,64 @@
-// use crate::ParachainInfo;
-// pub use codec::{Decode, Encode};
-// pub use cumulus_primitives_core::ParaId;
-// pub use frame_support::traits::Get;
-// pub use sp_runtime::traits::Convert;
-// pub use vln_primitives::{Asset, NetworkAsset};
-// pub use xcm::v0::{
-//     Junction::{GeneralKey, Parachain, Parent},
-//     MultiAsset,
-//     MultiLocation::{self, X1, X3},
-// };
+use crate::ParachainInfo;
+pub use codec::{Decode, Encode};
+pub use cumulus_primitives_core::ParaId;
+pub use frame_support::traits::Get;
+pub use sp_runtime::traits::Convert;
+pub use vln_primitives::{Asset, NetworkAsset};
+pub use xcm::v0::{
+    Junction::{GeneralKey, Parachain, Parent},
+    MultiAsset,
+    MultiLocation::{self, X1, X3},
+};
 
-// pub struct CurrencyIdConvert;
-// impl Convert<NetworkAsset, Option<MultiLocation>> for CurrencyIdConvert {
-//     fn convert(id: NetworkAsset) -> Option<MultiLocation> {
-//         match id {
-//             NetworkAsset::DOT => Some(X1(Parent)),
-//             NetworkAsset::ACA | NetworkAsset::AUSD => Some(native_currency_location(id)),
-//         }
-//     }
-// }
+pub struct CurrencyIdConvert;
+impl Convert<u32, Option<MultiLocation>> for CurrencyIdConvert {
```

Comment:
I like it how before was more explicit using the `NetwotkAsset` type instead of `u32` making it more explicit. Any special reason to have u32? I think even an alias would be better.

Reply:
The u32 comes from [here](https://github.com/valibre-org/vln-node/blob/38c5570546b65da72e1007dd159a05ccf0c1d7ab/runtime/src/lib.rs#L416), the type in pallet_assets and here should match, do you want me to type alias u32 here?

Reply:
mmm yeah I think helps readability